observability/logging/logging-6.1/log6x-clf-6.1.adoc

```diff
@@ -62,9 +62,9 @@ Filters:: Transform or drop log messages in the pipeline. Users can define filte
 
 Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
 
-application:: Selects logs from all application containers, excluding those in infrastructure namespaces such as `default`, `openshift`, or any namespace with the `kube-` or `openshift-` prefix.
+application:: Selects logs from all application containers, excluding those in infrastructure namespaces.
 
-infrastructure:: Selects logs from infrastructure components running in `default` and `openshift` namespaces and node logs.
+infrastructure:: Selects logs from infrastructure components running in `default`, `kube`, and `openshift` namespaces, or running in namespaces with the `kube-` or `openshift-` prefix, and node logs.
```

Comment:
Same comment as the other PR. 

To avoid combinations of multiple conjunctions, suggesting the following change for better clarity --

```suggestion
infrastructure:: Selects logs from nodes and from infrastructure components running in the following namespaces:
** `default`
** `kube`
** `openshift`   
** Containing the `kube-` or `openshift-` prefix
```

Reply:
This looks much better, thanks, updated.